packages/labs/src/components/tag-input/tagInput.tsx

```diff
@@ -34,6 +41,9 @@ export interface ITagInputProps extends IProps {
     /** React props to pass to the `<input>` element */
     inputProps?: HTMLInputProps;
 
+    /** Initial value of the `<input>` element */
```

Comment:
controlled value, not initial.

Reply:
Addressed in https://github.com/palantir/blueprint/pull/1967